apps/codeforafrica/src/payload/fields/richText.js

```diff
@@ -0,0 +1,57 @@
+import { deepmerge } from "@mui/utils";
```

Comment:
This PR and #516 both wants to add this field. May be we should add it on it's own first?